frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -311,38 +311,9 @@ const ManagePolicyPage = ({
     }
   );
 
-  const {
```

Comment:
teamPoliciesCount and isFetchingTeamCount was not being used (since we're using `teamPoliciesCountMergeInherited` and `isFetchingTeamCountMergeInherited`) so we can remove it